src/cursor/cur_metadata.c

```diff
@@ -30,15 +30,41 @@
 	    WT_CURSTD_VALUE_EXT);					\
 } while (0)
 
-#define	WT_MD_SET_KEY_VALUE(c, mc, fc) do {				\
-	(c)->key.data = (fc)->key.data;					\
-	(c)->key.size = (fc)->key.size;					\
-	(c)->value.data = (fc)->value.data;				\
-	(c)->value.size = (fc)->value.size;				\
-	F_SET((c), WT_CURSTD_KEY_EXT | WT_CURSTD_VALUE_EXT);		\
-	F_CLR((mc), WT_MDC_ONMETADATA);					\
-	F_SET((mc), WT_MDC_POSITIONED);					\
-} while (0)
+/*
+ * __curmetadata_setkv --
+ *	Copy the key/value pair from
```

Comment:
The comment is incomplete. It'd be good if it says something about manipulating the content.